packages/oats-runtime/src/make.ts

```diff
@@ -198,10 +196,26 @@ export function makeString(
   };
 }
 
-export function makeNumber(min?: number, max?: number): Maker<number, number> {
-  return (value: any) => {
-    if (typeof value !== 'number') {
-      return getErrorWithValueMsg('expected a number', value);
+export function makeNumber(
+  isInteger: boolean,
```

Comment:
is this necessary? cant't we have different makers for reals and integers?

Reply:
you're right, i didn't know how to make it dry at first, but now i found the solution 👍